.github/workflows/poetry-publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Python Poetry Build
+
+# the version on pypi is set in the `pyproject.toml` file
+# a tag that corresponds to the version should be pushed
+# whenever the version in that file is upgraded. The tag
+# will then trigger the deploy of the new package version
+# on pypi
+on:
+  push:
```

Comment:
Should we maybe only allow releases to be created from the main branch?

Reply:
Yes. We should do that. I just want to test first that it works.